### PR TITLE
single page result, compare page result, layout, style sheet (week April 23)

### PR DESCRIPTION
**Pull request by lucyzsunshine**

### CODE REVIEW

**Comment by ColeDCrawford** [comment]
@lucyzsunshine is this PR still needed?

**Comment by lucyzsunshine** [comment]
> @lucyzsunshine is this PR still needed?

let me check and close it once everything is ok

File: dle/search/templates/search/elastic/search.html [hunk 1]
```diff
@@ -97,6 +97,11 @@ <h5>Select one of three options:</h5>
             <input type="radio" id="search-type-semantic" name="search-type" value="knn">
             <label for="search-type-semantic">Semantic (Vector AKNN)</label>
           </div>
+          <div class="modal-body">
+            <h5>Tooltips in a modal</h5>
```
Comment (rcavezza) [inline dle/search/templates/search/elastic/search.html:101]: Is this test code? Does this do anything?

Comment (lucyzsunshine) [reply dle/search/templates/search/elastic/search.html:101]: just test. nothing here

**Comment by lucyzsunshine** [comment]
just for lucy test

**Pull request closed without merging**